create_db.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+#Give this script execution permision 
+# chmod +x create_db.sh
+
+# Delete database and user if they already exist
+dropdb tombolo
+dropdb tombolo_test
+dropuser tombolo
+dropuser tombolo_test 
+
+# Main database setup
+# Create a user and database
+createuser tombolo
+createdb -O tombolo tombolo -E UTF8
+psql -d tombolo -c "CREATE EXTENSION postgis;"
+psql -d tombolo -c "SET NAMES 'UTF8';"
+
+
+# Create DB tables and load initial fixtures
+psql -d tombolo -U tombolo < src/main/resources/sql/create_database.sql
+
+# Test database setu
+# Create a user and database
+createuser tombolo_test
+createdb -O tombolo_test tombolo_test -E UTF8
+psql -d tombolo_test -c "CREATE EXTENSION postgis;"
+psql -d tombolo_test -c "SET NAMES 'UTF8';"
+
+# Create DB tables and load initial fixtures
```

Comment:
I dont see a password being created for the user 'tombolo' and 'tombolo_test', but our gradle.properties file contains password, wont this fail when user would try to run the export or tests?

Reply:
This users are created without passwords, so they won't need to use them and the tests won't fail. If we think that it's better to change the instructions to creating the users with passwords, we could.

Reply:
if you have tested that tests and gradle task wont fail, then its okay